src/backend/src/routes/projects.routes.ts

```diff
@@ -9,21 +9,11 @@ const projectRouter = express.Router();
 projectRouter.get('/', ProjectsController.getAllProjects);
 projectRouter.get('/link-types', ProjectsController.getAllLinkTypes);
 projectRouter.get('/:wbsNum', ProjectsController.getSingleProject);
-projectRouter.post(
-  '/create',
-  intMinZero(body('crId')),
-  nonEmptyString(body('name')),
-  intMinZero(body('carNumber')),
-  nonEmptyString(body('summary')),
-  validateInputs,
-  ProjectsController.createProject
-);
-projectRouter.post(
-  '/edit',
-  intMinZero(body('projectId')),
+
+const projectValidators = [
   intMinZero(body('crId')),
   nonEmptyString(body('name')),
-  intMinZero(body('budget')),
+  body('budget').optional().isInt({ min: 0 }).default(0),
```

Comment:
I like this rather than a helper method, but budget shouldn't be optional on edit